test/core/compression/message_compress_test.cc

```diff
@@ -51,7 +51,7 @@ static void assert_passthrough(grpc_slice value,
   grpc_slice final;
   int was_compressed;
   const char* algorithm_name;
-  grpc_core::CompressionOptionsImpl options;
+  std::unique_ptr<CompressionOptions> options = grpc_core::MakeCompressionOptions(NULL);
```

Comment:
Please use `nullptr` instead of `NULL`.

Same for all of these tests.

Reply:
done